### DISCUSSION TITLE
No longer works.

### DISCUSSION THREAD

**Discussion by Max6025** [PetyXbron/minecraft-bot#28] [Q&A]
**Hello, Unfortunately it does not work any more on replit.com/ as well as on WIndows11 and Linux(Server 64gb) so I recommend deleting this project!**
![Screenshot 2024-01-11 205718](https://github.com/PetyXbron/minecraft-bot/assets/69343783/798d811d-3913-4e39-8229-bfd2b81bb71f)
![image](https://github.com/PetyXbron/minecraft-bot/assets/69343783/fff4d028-7ccc-46df-8ea3-499d6f5f1a01)

**Comment by PetyXbron** [comment]
This project is still working. You may face a bug, which seems you are having right now. But it is not good to create a discussion here for it. You'd be able to create a GitHub issue or message me on our Discord support server.
As the first screenshot shows, you have disallowed intents on your App page on the Discord Developers site. Check [this comment](https://github.com/PetyXbron/minecraft-bot/issues/27#issuecomment-1748054800) to resolve your issue. If it still occurs, contact me.
